src/main/java/me/lokka30/levelledmobs/listeners/EntityDeathListener.java

```diff
@@ -58,6 +63,24 @@ public void onDeath(@NotNull final EntityDeathEvent event) {
         if (lmEntity.getLivingEntity().getKiller() != null && main.placeholderApiIntegration != null)
             main.placeholderApiIntegration.putPlayerOrMobDeath(lmEntity.getLivingEntity().getKiller(), lmEntity);
 
+        if(lmEntity.isLevelled() || main.rulesManager.getRule_UseCustomDropsForMob(lmEntity).useDrops){
+            long chunkKey = Utils.getChunkKey(lmEntity);
+            if(!main.entityDeathInChunkCounter.containsKey(chunkKey)){
+                main.entityDeathInChunkCounter.put(chunkKey,new IndexedTreeSet<>());
+            }
+            main.entityDeathInChunkCounter.get(chunkKey).add(new MutablePair<>(new Timestamp(System.currentTimeMillis()),lmEntity));
+        }
+
+        int numberOfEntityDeathInChunk=Utils.getNumberOfEntityDeathInChunk(lmEntity,main,main.maximumCoolDownTime);
+        // Only send message for maximum threshold and cool down time
+        // This is enabled by default
+        if(numberOfEntityDeathInChunk>main.maximumDeathInChunkThreshold) {
+            if(lmEntity.getLivingEntity().getKiller() != null && lmEntity.getLivingEntity().getKiller() instanceof Player &&
+                main.settingsCfg.getBoolean("exceed-kill-in-chunk-message",true))
```

Comment:
space after the comma

Reply:
this statement needs to be wrapped in curly braces `{ }` since the content goes over multiple lines